src/jbi/models.py

```diff
@@ -16,7 +17,8 @@ class Action(YamlModel):
     Action is the inner model for each action in the configuration file"""
 
     action: str = "src.jbi.whiteboard_actions.default"
-    contact: str
+    # TODO: Remove the tbd literal option when all actions have contact info
```

Comment:
```suggestion
    # TODO: Remove the tbd literal option when all actions have contact info # pylint: disable=fixme
```

I think this could solve the error? Unless we want a global change to allow ToDo's